### ISSUE TITLE
Single clicking on T-clock does not close Win10 calendar

### ISSUE THREAD

**Issue by Fanolian** [White-Tiger/T-Clock#10]
OS: Win10 Pro 64bit
T-Clock Redux: 2.4.0 build 341 or 351(beta/rc) 

Steps to reproduce:
1. (To mimic Win10 behaviour) Properties > Mouse: Set single left/right click to open calendar.
2. Single click on T-clock to pop out Win10's calendar.
3. Try to click again on T-clock to close calendar.

Actual result:
Calendar opens again. I have to click outside T-clock or calendar to close it.

Expected result:
Calendar closes when I click on the clock, a la Win10's default behaviour.

Note:
The calendar closes correctly on clicking if I enable "Restore previous calendar" in Properties > Miscellaneous.

**Comment by White-Tiger** [comment]
will try to reproduce, but it should work *(and did last time I tried)*.. unless your Windows isn't English. Though your other issue suggests it is.

**Comment by White-Tiger** [comment]
I couldn't reproduce the "opens again" case but had a different issue with the calendar not closing at all...
I've made some adjustments in commit 69bdb45 which should fix both. *(I hope)*

**Fun fact:** the only Windows calendar that really worked was the Vista one *(which also added the calendar in first place)*... every calendar since then is bugged by Windows itself. Basically it might also happen without T-Clock that your 2nd click re-opens the calendar *(disappears then opens again)*
T-Clock is partially even more reliable in closing the calendar than Windows. I had to add some ugly hacks for this reason but they helped to reduce "false-positives"
